client/js/components/button/NavButton.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <!--The attribute data-responsive-menu-helper is used as a hook to append an EventListener in GlobalEventlisteners.js-->
+  <button
+    type="button"
+    class="btn btn--blank font-size-medium"
+    aria-haspopup="true"
+    aria-expanded="false"
+    data-responsive-menu-helper>
+    <a class="nav-toggle">
```

Comment:
I guess placing an `a` element within a `button` element is invalid Html.
Also, the styles for `.nav-toggle` are located inside the project overrides - i'd suggest to also place them in the core, maybe in `scss/components/_<component-name>.scss`.

Reply:
done https://github.com/demos-europe/demosplan-core/pull/1389/commits/692686822ba44f7c25e4e7bb57ddbbcf42bcf6af